frontend/rolecall/src/app/performance/performance-editor.component.ts

```diff
@@ -21,8 +22,18 @@ import {CAST_COUNT} from 'src/constants';
 })
 export class PerformanceEditor implements OnInit, OnDestroy, AfterViewChecked {
 
+  constructor(private performanceAPI: PerformanceApi,
```

Comment:
Let's move this back down as the first function in the file, so it's not mixed in with the properties

Reply:
Done.